monai/apps/detection/transforms/dictionary.py

```diff
@@ -687,10 +697,81 @@ def inverse(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, Nd
         return d
 
 
+class ClipBoxToImaged(MapTransform, InvertibleTransform):
+    """
+    Dictionary-based wrapper of :py:class:`monai.apps.detection.transforms.array.ClipBoxToImage`.
+
+    Clip the bounding boxes and the associated labels/scores to makes sure they are within the image.
+    There might be multiple keys of labels/scores associated with one key of boxes.
+
+    Args:
+        box_keys: The single key to pick box data for transformation. The box mode is assumed to be ``StandardMode``.
+        label_keys: Keys that represents the lables corresponding to the ``box_keys``.
```

Comment:
Do you support multiple `label_keys`?
I feel it only supports single key as `box_keys`.

Thanks.

Reply:
Yes, it supports multiple label_keys. I put an example in docstring.